doc/content/examples/blog/_index.md

```diff
@@ -0,0 +1,4 @@
+---
+title: Blog
+type: posts
```

Comment:
We may want to allow `post` here as well, just so this reads more naturally. Currently, it's basically "the type of this page is the same as if it were in the posts category", but would be more logical to say "the type of this page is a post".